.editorconfig

```diff
@@ -11,6 +11,9 @@ root = true
 indent_style = space
```

Comment:
**TLDR;**  No this is project based editor setup, the setting itself is personal flavoured. I find it a good start to consistent written code. I'll keep this where it is.


**Extra reading material:**
It is a common setting and one I would like to defend to get a some sort of first basic consistency in the project files. Also commonly used together with linting and formatting editor plugins.

If one preferes to use tabs and other spaced tabs, I think that is a first step toward inconsistent written code. Even with this setting  already in place I have seen this setting (and other linting rules) been ignored at several places in the code.

For this reason I was thinking about implementing lint-staged at some point, because often enough project based conventions and settins are set aside to just get things done faster. With lint-staged, contributers have to, after implementing their own flavoured fix, adhere to the project's conventions and refactor their working code into the desired format.